site/pages/institution-contact-en.hbs

```diff
@@ -0,0 +1,49 @@
+---
+{
+	"title": "Contact [Institution name]",
+	"language": "en",
+	"altLangPrefix": "institution-contact",
+	"pageType": "institution",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" },
+		{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+	],
+	"dateModified": "2017-09-29",
+	"share": "true"
+}
+---
+<div class="row">
+<div class="col-sm-7 pull-left">
+{{>byline}}
+      <p>[Institution name] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
+</div>
+	<div class="clr-rght-sm clr-rght-md clr-rght-lg"></div>
+	<div class="col-sm-5 pull-right">
```

Comment:
Can you do that block without the "pull-". Try to use instead "col-sm-pull-" and "col-sm-push-"